src/components/products/products-styles.js

```diff
@@ -0,0 +1,119 @@
+import styled from "styled-components"
+import dimensions from "../../style/dimensions"
+
+export const ProductPageHeading = styled.div`
+  font-size: calc(90px + (90 - 32) * ((100vw - 300px) / (1440 - 300)));
```

Comment:
Now that we have a typography file we should just import H1 from the typography-styles.js